packages/gestalt/src/Typeahead.js

```diff
@@ -54,8 +55,9 @@ type Props = {|
 const TypeaheadWithForwardRef: React$AbstractComponent<Props, HTMLInputElement> = forwardRef<
   Props,
   HTMLInputElement,
->(function Typeahead(props, ref): Node {
-  const {
+>(function Typeahead(
+  {
```

Comment:
can we keep as it was? ForwardRefs are hard to read and all other components keep the same structure of the destructuring outside. Just for consistency across cmps.